examples/inverse/evoked_ers_source_power.py

```diff
@@ -31,37 +32,37 @@
 
 # %%
 # Reading the raw data and creating epochs:
+
 data_path = somato.data_path()
 subject = '01'
+subjects_dir = data_path / 'derivatives' / 'freesurfer' / 'subjects'
 task = 'somato'
 raw_fname = (data_path / 'sub-{}'.format(subject) / 'meg' /
              'sub-{}_task-{}_meg.fif'.format(subject, task))
 
 # crop to 5 minutes to save memory
 raw = mne.io.read_raw_fif(raw_fname).crop(0, 300)
 
-# We are interested in the beta band (12-30 Hz)
+# we are interested in the beta band (12-30 Hz)
 raw.load_data().filter(12, 30)
 
-# The DICS beamformer currently only supports a single sensor type.
-# We'll use the gradiometers in this example.
+# the DICS beamformer currently only supports a single sensor type,
+# we'll use the gradiometers in this example
```

Comment:
```suggestion
# The DICS beamformer currently only supports a single sensor type.
# We'll use the gradiometers in this example.
```
I briefly checked in with @drammock et al. on this and there is no standard to comments in MNE-Python. The consensus seems to be that if comments are full sentences, they should have proper punctuation etc. If comments become large chunks, they should be move to the Markdown parts.
So here, I would not change what the example did before. 
I also agree with the comment above that this also adds a lot of clutter to the PR.